src/Chainweb/CutDB.hs

```diff
@@ -730,17 +733,20 @@ blockDiffStream :: MonadIO m => CutDb cas -> S.Stream (Of (Either BlockHeader Bl
 blockDiffStream db = cutStreamToHeaderDiffStream db $ cutStream db
 
 cutHashesToBlockHeaderMap
-    :: PayloadCas cas
+    :: forall cas
```

Comment:
This quantification should not be necessary; you aren't using the variable within the function anywhere that I can see.